src/DynamoPublish/Models/PublishModel.cs

```diff
@@ -302,11 +305,52 @@ internal void Authenticate()
 
             State = UploadState.Uploading;
 
-            IReachHttpResponse result;
+            IReachHttpResponse result = null;
+            IEnumerable<Workspace> wss = null;
```

Comment:
Is there a reason these need to be initialized as false? There should not be a code path where they are empty...